src/client.js

```diff
@@ -130,7 +130,8 @@ export default class Client {
     });
 
     this.on('sasl:failure', (err) => {
-      this.logger.error('Authentication failed connecting to streaming service', err);
+      const errMessage = `Authentication failed connecting to streaming service\nchannelId: ${this.config.channelId}`;
```

Comment:
It looks like a minor typo in `service\nchannelId:` to `service\channelId:`.

Reply:
That's a newline character :)

Reply:
Oof, you are right. For some reason I was reading it like you were trying to do a normal `\`